src/saving/SaveListItem.cs

```diff
@@ -138,6 +146,7 @@ public override void _Process(float delta)
 
         // General info
         version.Text = save.Info.ThriveVersion;
+        versionWarning.Visible = save.Info.ThriveVersion != Constants.Version;
```

Comment:
Making `save.Info.ThriveVersion != Constants.Version` into a property of this class would be nice.